doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+
+    curl ... -H 'Content-Type: application/json'
+
+This request will send data in well defined JSON format.
+
+[Top](#top)
+ 
+## 4. Connection
+ 
+ This header is used by both requests and responses.
+ Orion is not currently doing anything with this header (apart from checking if its value is `close` to print a log trace).
+	
+    Connection: Keep-Alive
+    	
+[Top](#top)	
+  
+## 5. Expect
+
+Expect HTTP header is used in requests.
+It Indicates that behaviour of particular server are required by the client.
+It is typically used when sending a large request body. We expect the server to return back `100 Continue` HTTP status if it can handle the request, or `417 Expectation Failed` if not.
+
+    Expect: 100-continue
+	
+[Top](#top)	
+	
+## 6. Host
+
+This HTTP header is used in Orion responses.
+Orion includes this header in outgoing request (notifications and forwarded updates/queries).This header provides the port number of notification receiver. 	
+	
+    Host: localhost:1028
+		
+[Top](#top)
+
+## 7. Origin
+
+Origin is used in requests.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md).
+	
+    Origin: http://www.example-social-network.com
+		
+[Top](#top)	
+  
+## 8. User-Agent
+
+User-Agent HTTP header is used in notifications and forwarded requests.
+It describes which version of Orion we are using along with its transfer library.
+It gives the string that identifies the user agent.
+
+    User-Agent: orion/2.2.0-next libcurl/7.29.0     (in notifications and forwarded requests)
+
+[Top](#top)		
+		
+## 9. X-Forwarded-For
+
+It is a request HTTP header in Orion.
+Source IP of the HTTP request associated to the transaction, except if the request includes X-Forwarded-For header which overrides the former IP.
+X-Real-IP and X-Forwarded-For (used by a potential proxy on top of Orion) overrides IP.
+X-Real-IP takes preference over X-Forwarded-For, if both appear.
+	
+    X-Forwarded-For: 129.78.138.66
+	
+[Top](#top) 
+	
+## 10. X-Real-IP
+
+Source IP of the HTTP request associated to the transaction, except if the request includes X-Real-IP which overrides X-Forwarded-For and source IP.
+X-Real-IP and X-Forwarded-For (used by a potential proxy on top of Orion) overrides IP.
+X-Real-IP takes preference over X-Forwarded-For, if both appear.
+	
+[Top](#top)
+
+## 11.  Access-Control-Allow-Origin
+
+It is an optional header used in Orion responses.  
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-allow-origin).
+    
+    Access-Control-Allow-Origin: *
+		
+[Top](#top)
+
+
+## 12. Access-Control-Allow-Headers
+
+It is a response HTTP header in Orion.
```

Comment:
```suggestion
It is header used in outgoing HTTP responses HTTP sent by Orion.
```

What do you think about this style of describing which role takes a given header? I think that the usage of incoming/outgoing can be very clarifying.

If you think is a good idea, I'd suggest to review the overall document and apply it where it applies.

Reply:
Yes. This descriptions is looking good.
So I'm modifying the doc file with the same.

Reply:
Fixed in a427bd9